tests/test_enterprise/test_signals.py

```diff
@@ -625,3 +626,61 @@ def test_update_catalog_with_existing_service_catalog(self, api_client_mock):
             enabled_course_modes=enterprise_catalog.enabled_course_modes,
             publish_audit_enrollment_urls=enterprise_catalog.publish_audit_enrollment_urls
         )
+
+    @mock.patch('enterprise.signals.EnterpriseCatalogApiClient')
+    def test_update_enterprise_query(self, api_client_mock):
```

Comment:
Might be worth adding a docstring to explain what this test does? Especially the part about also ensuring the changes are synced to the enterprise catalog service as that's easy to miss